build/ferretdb.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=FerretDB Database
+Documentation=https://docs.ferretdb.io/
+Wants=network-online.target
+After=network-online.target
+
+[Service]
```

Comment:
I see we don't set `user` and `group` here. I thought, typically, services have it - to ensure that things are not run from `root` and that permissions are separated. I guess, it's not top priority?

Reply:
We don't know if we want it or not yet